mixer/cmd/build.go

```diff
@@ -788,3 +802,23 @@ func checkFormatBump(b *builder.Builder) error {
 
 	return nil
 }
+
+// parseRepoURLOverrides parses the --from-repo-url and --to-repo-url arguments
+// to create a map of repo keys with URL values.
+func parseRepoURLOverrides(repoURLs []string) (map[string]string, error) {
```

Comment:
Isn't there are facility provided by Cobra to parse a list of args like this?

Reply:
You are right, the `StringToString` Pflags type can store flag arguments into a map[string]string. The flag format will change to the following:

`--from-repo-url <repo>=<url>`

Our current version of Pflags does not support this feature, so we need to bump the version.

Reply:
Upgrading Cobra from 0.0.1 to 0.0.5 enables the required flags.